db/migrate/20130128124852_acts_as_taggable_on_migration.rb

```diff
@@ -1,4 +1,5 @@
 class ActsAsTaggableOnMigration < ActiveRecord::Migration
```

Comment:
If the migrations don't need to be run, can we just delete them and add them as a separate pending PR? Otherwise, the migration entry would've been made for this dummy migration.